docs/UserGuide.md

```diff
@@ -130,25 +130,36 @@ Edits an existing order in the order list.
 Format: `edito INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/ITEM_NAME] [q/QUANTITY]`
 
 * Edits the order at the specified `INDEX`.
+* This feature is case-insensitive.
 * The index refers to the index number shown in the displayed order list.
 * The index **must be a positive integer** 1, 2, 3, …​
-* Every field is optional, except for `i/ITEM_NAME` and `q/QUANTITY`. Both fields need to be present to update
-  the customer's ordered items.
-* When editing items in the order list, this feature is case-insensitive.
-* You can only edit the order list to consist of items that exists in your inventory. <br> 
+* Every field is optional, but if you were to include `i/ITEM_NAME`, you must also include 
+  `q/QUANTITY`. Both fields need to be present to update the customer's ordered items.
+* You can only edit the customer's ordered items list to consist of items that exists in your inventory. <br> 
   e.g. If your inventory does not have `Apples`, then you cannot edit your customer's order list to have `Apples`.
 * Editing an item that does not exist in your customer's order list, but exists in your inventory 
-  will add the item to the order list.
-* Setting an ordered item quantity to `0` will remove the item from the order list.
+  will add the item to the order list. <br> e.g. Your inventory has `Apples` and `Bananas`. <br>
+  Right now, the third customer's ordered item list only has `Apples`.
+  `edito 3 i/Bananas q/3` will add `3` `Bananas` to the customer's ordered item list.
+* Setting `q/0` to a customer's ordered item list will remove the item from the list.
```

Comment:
Might be good to elaborate more, and give a relevant example

Reply:
Ok can! 👍